File: backend/pois.js
// Points of Interest — WUF13 Venue
// Extracted from venue Masterplan (Scale 1:2000)
// Individual meeting room numbers inside Area A must be confirmed on-site.

const POIS = [
  // ─── Entrances / Exits ───────────────────────────────────────────────────
  { code: 'ENTRANCE_MAIN',          name_en: 'Main Entrance',                   name_az: 'Əsas Giriş' },
  { code: 'ENTRANCE_WUF',           name_en: 'WUF Entrance',                    name_az: 'WUF Girişi' },
  { code: 'ENTRANCE_STAFF',         name_en: 'Staff Entrance / Exit',           name_az: 'Personal Giriş/Çıxışı' },
  { code: 'ENTRANCE_INTL_SERVICE',  name_en: 'International Services Entrance', name_az: 'Beynəlxalq Xidmət Girişi' },
  { code: 'EXIT_NORTH',             name_en: 'Exit (North)',                    name_az: 'Çıxış (Şimal)' },
  { code: 'EXIT_SOUTH',             name_en: 'Exit (South)',                    name_az: 'Çıxış (Cənub)' },

  // ─── Area A — Registration & Meeting Rooms ───────────────────────────────
  { code: 'AREA_A_REGISTRATION',    name_en: 'Registration Area',               name_az: 'Qeydiyyat Zonası' },
  { code: 'AREA_A_MEETING_ROOMS',   name_en: 'Meeting Rooms',                   name_az: 'Konfrans Zalları' },
  { code: 'AREA_A_BILATERAL_HUB',   name_en: 'Bilateral Hub',                   name_az: 'Konfrans Mərkəzi' },

  // ─── Area B — Main Arena ─────────────────────────────────────────────────
  { code: 'AREA_B_MEDIA_HUB',       name_en: 'Media Hub',                       name_az: 'Media Mərkəzi' },
  { code: 'AREA_B_ROUND_TABLE',     name_en: 'Round Table Rooms',               name_az: 'Dairəvi Masa Otaqları' },
  { code: 'AREA_B_SPECIAL_SESSIONS',name_en: 'Special Sessions',                name_az: 'Xüsusi Sessiyalar' },
  { code: 'AREA_B_DIALOGUE_ROOMS',  name_en: 'Dialogue Rooms',                  name_az: 'Dialoq Otaqları' },

  // ─── Area C — Expo ───────────────────────────────────────────────────────
  { code: 'AREA_C_EXPO',            name_en: 'Expo Centre',                     name_az: 'Ekspo Mərkəzi' },

  // ─── Area D — Ceremony & VIP ─────────────────────────────────────────────
  { code: 'AREA_D_CEREMONY_HALL',   name_en: 'Ceremony Hall',                   name_az: 'Tədbir Zalı' },
  { code: 'AREA_D_VVIP_HALL',       name_en: 'VVIP Hall',                       name_az: 'VVİP Zalı' },

  // ─── Transport / Drop-off ────────────────────────────────────────────────
  { code: 'TRANSPORT_DROPOFF_NORTH',name_en: 'Drop-off Zone North',             name_az: 'Minmə/Düşmə Zonası Şimal' },
  { code: 'TRANSPORT_DROPOFF_SOUTH',name_en: 'Drop-off Zone South',             name_az: 'Minmə/Düşmə Zonası Cənub' },
  { code: 'TRANSPORT_BUS_STAGING',  name_en: 'Bus Staging Area',                name_az: 'Ehtiyat Avtobus Yeri' },
  { code: 'TRANSPORT_EV_CHARGING',  name_en: 'EV Charging Area',                name_az: 'Elektromobil Şarj Zonası' },
  { code: 'PARKING_VVIP',           name_en: 'V/VIP Parking',                   name_az: 'V/VİP Dayanacaq' },

  // ─── Services ────────────────────────────────────────────────────────────
  { code: 'SERVICE_WC',             name_en: 'Toilets',                         name_az: 'Sanitar Qovşağı' },
  { code: 'SERVICE_DOG_RELIEF',     name_en: 'Dog Relief Area',                 name_az: 'İt üçün Tualet Sahəsi' },
  { code: 'SERVICE_AMBULANCE',      name_en: 'Ambulance / First Aid',           name_az: 'Təcil Yardım' },
  { code: 'STADIUM',                name_en: 'Stadium',                         name_az: 'Stadion' },
];

module.exports = POIS;

File: frontend/soundstep/lib/data/pois.dart
class Poi {
  final String code;
  final String nameEn;
  final String nameAz;

  const Poi({required this.code, required this.nameEn, required this.nameAz});

  String name(String lang) => lang == 'az' ? nameAz : nameEn;
}

const List<Poi> kPois = [
  Poi(code: 'ENTRANCE_MAIN',          nameEn: 'Main Entrance',                   nameAz: 'Əsas Giriş'),
  Poi(code: 'ENTRANCE_WUF',           nameEn: 'WUF Entrance',                    nameAz: 'WUF Girişi'),
  Poi(code: 'ENTRANCE_STAFF',         nameEn: 'Staff Entrance / Exit',           nameAz: 'Personal Giriş/Çıxışı'),
  Poi(code: 'ENTRANCE_INTL_SERVICE',  nameEn: 'International Services Entrance', nameAz: 'Beynəlxalq Xidmət Girişi'),
  Poi(code: 'EXIT_NORTH',             nameEn: 'Exit (North)',                    nameAz: 'Çıxış (Şimal)'),
  Poi(code: 'EXIT_SOUTH',             nameEn: 'Exit (South)',                    nameAz: 'Çıxış (Cənub)'),
  Poi(code: 'AREA_A_REGISTRATION',    nameEn: 'Registration Area',               nameAz: 'Qeydiyyat Zonası'),
  Poi(code: 'AREA_A_MEETING_ROOMS',   nameEn: 'Meeting Rooms',                   nameAz: 'Konfrans Zalları'),
  Poi(code: 'AREA_A_BILATERAL_HUB',   nameEn: 'Bilateral Hub',                   nameAz: 'Konfrans Mərkəzi'),
  Poi(code: 'AREA_B_MEDIA_HUB',       nameEn: 'Media Hub',                       nameAz: 'Media Mərkəzi'),
  Poi(code: 'AREA_B_ROUND_TABLE',     nameEn: 'Round Table Rooms',               nameAz: 'Dairəvi Masa Otaqları'),
  Poi(code: 'AREA_B_SPECIAL_SESSIONS',nameEn: 'Special Sessions',                nameAz: 'Xüsusi Sessiyalar'),
  Poi(code: 'AREA_B_DIALOGUE_ROOMS',  nameEn: 'Dialogue Rooms',                  nameAz: 'Dialoq Otaqları'),
  Poi(code: 'AREA_C_EXPO',            nameEn: 'Expo Centre',                     nameAz: 'Ekspo Mərkəzi'),
  Poi(code: 'AREA_D_CEREMONY_HALL',   nameEn: 'Ceremony Hall',                   nameAz: 'Tədbir Zalı'),
  Poi(code: 'AREA_D_VVIP_HALL',       nameEn: 'VVIP Hall',                       nameAz: 'VVİP Zalı'),
  Poi(code: 'TRANSPORT_DROPOFF_NORTH',nameEn: 'Drop-off Zone North',             nameAz: 'Minmə Zonası Şimal'),
  Poi(code: 'TRANSPORT_DROPOFF_SOUTH',nameEn: 'Drop-off Zone South',             nameAz: 'Minmə Zonası Cənub'),
  Poi(code: 'TRANSPORT_BUS_STAGING',  nameEn: 'Bus Staging Area',                nameAz: 'Ehtiyat Avtobus Yeri'),
  Poi(code: 'TRANSPORT_EV_CHARGING',  nameEn: 'EV Charging Area',                nameAz: 'Elektromobil Şarj Zonası'),
  Poi(code: 'PARKING_VVIP',           nameEn: 'V/VIP Parking',                   nameAz: 'V/VİP Dayanacaq'),
  Poi(code: 'SERVICE_WC',             nameEn: 'Toilets',                         nameAz: 'Sanitar Qovşağı'),
  Poi(code: 'SERVICE_DOG_RELIEF',     nameEn: 'Dog Relief Area',                 nameAz: 'İt üçün Tualet Sahəsi'),
  Poi(code: 'SERVICE_AMBULANCE',      nameEn: 'Ambulance / First Aid',           nameAz: 'Təcil Yardım'),
  Poi(code: 'STADIUM',                nameEn: 'Stadium',                         nameAz: 'Stadion'),
];
